omniscidb/L0Mgr/L0MgrNoL0.cpp

```diff
@@ -133,6 +133,14 @@ unsigned L0Manager::getMinEUNumForAllDevices() const {
   CHECK(false);
 }
 
+bool L0Manager::hasSharedMemoryAtomicsSupport() const {
+  CHECK(false);
```

Comment:
Could you please add `return` statements in this and similar implementations? I know this `CHECK` leads to `abort` call and the return statement would be a dead code but compilers complain about missing returns and on Windows, it is treated as an error, not just a warning.

Reply:
Right, thanks for reminding me!